### PR TITLE
Fix some minor typos

### PR DESCRIPTION
**Pull request by dextermb**
I've fixed the typos that have been previously discussed. Let me know if you notice anymore.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@langleyfoxall/react-dynamic-data-table",
-  "version": "2.3.0",
+  "version": "2.3.1",
```
Comment (DivineOmega) [inline package.json]:
```suggestion
  "version": "2.3.0",
```

There is not need to update the version, as this is not a bug fix, new feature, or breaking cahnge.

Comment (dextermb) [reply package.json]: NPM won't show the read me unless you publish, you can't publish the same version twice

Comment (DivineOmega) [reply package.json]: Fair enough. Let's bump the version as you suggested.

Comment (dextermb) [reply package.json]: It's just annoying that the readme isn't pulled from the repo, such as composer.

**Pull request merged**